app/components/public/side-menu.js

```diff
@@ -2,7 +2,7 @@ import Component from '@ember/component';
 import { computed } from '@ember/object';
 
 export default Component.extend({
-  shouldShowCallforSpeakers: computed(function() {
-    return this.get('event.isSessionsSpeakersEnabled') && (this.get('event.speakersCall.privacy') === 'public');
+  shouldShowCallforSpeakers: computed(async function() {
```

Comment:
This is incorrect. Don't perform async tasks within computed properties. Instead, do this on `didInsertElement` and set `shouldShowCallforSpeakers`